bundled_openapi.json

```diff
@@ -5234,7 +5234,8 @@
                                   "RED",
                                   "GREEN",
                                   "YELLOW",
-                                  "ORANGE"
+                                  "ORANGE",
+                                  "GRAY"
```

Comment:
Gray doesn't appear to be in our list of allowed colors...  Am I missing something?

Reply:
So this color can't be used by a user, it's only used by the backend when there was some error communicating with the external watchlist.

Reply:
OH neat... I didn't know that.

Reply:
for the record, GRAY color is set [here](https://github.com/tractionguest/guest-server/blob/5954ce73c153ee0c3f2ff2a5c3940dec4f90cde6/app/models/integration.rb#L149-L154)